aspnetcore/fundamentals/servers/kestrel/sample/Properties/launchSettings.json

```diff
@@ -0,0 +1,28 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:8096/",
+      "sslPort": 0
+    }
+  },
+  "profiles": {
+    "IIS Express": {
```

Comment:
I tried to remove IIS Express profile but it comes back as soon as I rebuild.
@Rick-Anderson Do you know how to remove IIS Express profile so it stays removed?